swarm/tools/validation/validators/prompts.py

```
@@ -0,0 +1,127 @@
+
+LEGACY: This check is skipped if .claude/agents/ directory does not exist.
+The new architecture uses swarm/config/agents/ for agent configuration instead.
+
+Checks for presence of these required headings after frontmatter:
+- ## Inputs (or ## Input)
+- ## Outputs (or ## Output)
+- ## Behavior
+"""
+
+import re
+from typing import Any, Dict, List
```

Ruff (0.14.13)

`typing.Dict` is deprecated, use `dict` instead

(UP035)